### PR TITLE
Added first post in the agility/scrum series

### PR DESCRIPTION
**Pull request by davidaayers**

### CODE REVIEW

File: _posts/2016-08-30-agility.markdown [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+author: david
+bigimg: /img/train-tracks.jpg
+---
+# Everything You Know About Building Software Is Wrong
+
+It's not your fault. Software is complicated, and no one understands that making something that looks and feels *easy* is actually *hard*.
+
+The common metaphor is that building software is like building a house. You have an architect draw up the plans, you give those plans to some builders, who build everything exactly according to the plan, and at the end, your house is delivered exactly as you wanted it, on time and on budget.
+
+Which, of course, as anyone that has ever built a house can tell you, silly. Sub-contractors don't show up. Tile choices are out of stock. Rain delays! Cost overruns galore!
```
Comment (jrdaniels) [inline _posts/2016-08-30-agility.markdown]: ", is silly."

Should there be an "is" there?

Comment (davidaayers) [inline _posts/2016-08-30-agility.markdown]: Added an "is", but in a different place.

File: _posts/2016-08-30-agility.markdown [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+The other benefit is risk reduction. When we do a giant release two or three times a year, the amount of technical changes that occur all at once are huge! Changing a hundred things at one time is much riskier than changing ten things at one time, and even figuring out where a bug is becomes much easier when you have only changed ten things, and you made those ten changes over the last couple of weeks (instead of over the last six months). The context is much clearer for things that we just did last week versus things we did months ago, so it's easier to address any problems that might arise quickly.
+
+# Whew. That's A Lot!
+
+Thanks for sticking around to the end. This is the first in a series of article that introduces people to how we build software, and more importantly, *why* we build it the way we do.
```
Comment (jrdaniels) [inline _posts/2016-08-30-agility.markdown]: make article plural

Comment (davidaayers) [inline _posts/2016-08-30-agility.markdown]: Fixed and pushed!

File: _posts/2016-08-30-agility.markdown [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+And for us at TCS, there is a parallel that Brad Schneider has pointed out; there are echoes of our Foundation Principles in many of these agile principles. Just a few examples:
+
+|Agile Principle|Foundation Principle|
+|---------------|--------------------|
+|Business people and developers must work together daily throughout the project.|Communication IS Leadership|
+|Build projects around motivated individuals. Give them the environment and support they need, and trust them to get the job done.|1 = 3|
+|The most efficient and effective method of conveying information to and within a development team is face-to-face conversation.|Communication IS Leadership|
+|At regular intervals, the team reflects on how to become more effective, then tunes and adjusts its behavior accordingly.|Intuition Does Not Come To An Unprepared Mind|
+
+Adopting these agile principles in our software development practices was a natural fit for TCS; our culture already had many of these behaviors already baked in.
```
Comment (JohnThrailkill) [inline _posts/2016-08-30-agility.markdown]: Drop the second "already"

File: _posts/2016-08-30-agility.markdown [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+# Software Shouldn't Be Inventory
+
+So, let's bring it all together. Using the old ways of making software, we go through our waterfall process, and many months (or years!) later, the software is ready to launch. Imagine that during that time, we are building up an inventory of features. Those features only begin to accrue value when they are actually put in the hands of our users. So, the longer there is between the time a developer builds a feature and a user is using it, the longer that inventory is just sitting on the shelf. Just like in our distribution center, inventory that's on a shelf is a *cost* for us, rather than delivering *value* for us.
```
Comment (JohnThrailkill) [inline _posts/2016-08-30-agility.markdown]: I would change "accrue value" to "deliver value" just to be more accessible to everyone.

**Comment by JohnThrailkill** [comment]
really excellent 👍

**Comment by davidaayers** [comment]
@JohnThrailkill comments addressed.

**Comment by jrdaniels** [comment]
I'm not sure what makes me more excited:  the blog, or that @JohnThrailkill is commenting on pull requests.... 👏

**Comment by davidaayers** [comment]
Following Ed's suggestions, I broke the first article into two different articles, and did a bit of cleanup.

This is accessible now on the web (internal to TCS):

http://labdaas1/

**Comment by jrdaniels** [comment]
👍

**Comment by JohnThrailkill** [comment]
Looks great to me!

John Thrailkill | Executive Vice President of IT & Business Development | The Container Store
972-538-6649 phone | 972-538-7649 fax
containerstore.com<http://www.containerstore.com/#containerstore.com> |
 What We Stand For<http://standfor.containerstore.com/#What%20We%20Stand%20For>

**Pull request merged**